deny.toml

```diff
@@ -0,0 +1,25 @@
+[graph]
+all-features = true
+
+[licenses]
+allow = [
```

Comment:
Comparing your list from [this resource](https://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses), it looks good to me.
Based on the list we can add more, for example:
` ` ` 
"MPL-2.0",
"Unlicense",
` ` ` 
but might be is not needed for now